### ISSUE TITLE
bzip2 shared libraries error

### ISSUE THREAD

**Issue by tombair** [conda/conda#3632]
I try to do an install:

bash Miniconda-latest-Linux-x86_64.sh -b
PREFIX=/Users/tbair/miniconda2
installing: _cache-0.0-py27_x0 ...
bzip2: error while loading shared libraries: libbz2.so.1.0: cannot open shared object file: No such file or directory
tar: Child returned status 127
tar: Error is not recoverable: exiting now

If I crudely remove the unset LD_LIBRARY_PATH and size check it works fine:

bash Miniconda-latest-Linux-x86_64.sh -b -f
PREFIX=/Users/tbair/miniconda2
WARNING: md5sum mismatch of tar archive
expected: 4a6452c9986d85aaa7cf382d4e7f6f31
     got: 38a3df3b885a887867afead47daba493  -
installing: _cache-0.0-py27_x0 ...
installing: python-2.7.11-0 ...
installing: conda-env-2.4.5-py27_0 ...
installing: openssl-1.0.2g-0 ...
installing: pycosat-0.6.1-py27_0 ...
installing: pyyaml-3.11-py27_1 ...
installing: readline-6.2-2 ...
installing: requests-2.9.1-py27_0 ...
installing: sqlite-3.9.2-0 ...
installing: tk-8.5.18-0 ...
installing: yaml-0.1.6-0 ...
installing: zlib-1.2.8-0 ...
installing: conda-4.0.5-py27_0 ...
installing: pycrypto-2.6.1-py27_0 ...
installing: pip-8.1.1-py27_1 ...
installing: wheel-0.29.0-py27_0 ...
installing: setuptools-20.3-py27_0 ...
Python 2.7.11 :: Continuum Analytics, Inc.
creating default environment...
installation finished.

Not sure if that is ok or dumb and would like a clean install as it is part of a larger bcbio install

**Comment by bernt-matthias** [comment]
Same here. On my system (CentOS6.8) tar is not working if LD_LIBRARY_PATH is not set. I can remove the LD_LIBRARY_PATH and everything works.

I'm installing conda via galaxy 17.10.

**Comment by bernt-matthias** [comment]
The problem is that our cluster uses environment modules (http://modules.sourceforge.net/) to provide software (basically by setting and unsetting PATHs...). Because python needs bz2 support for galaxy I needed to load bzip2 as module since the dynamic library was not available otherwise. This module also provides a bunzip2 binary which led to a clash when the LD_LIBRARY_PATH was unset during conda installation (the module's bunzip2 still wanted to have the dynamic library and the systems bunzip2 was later in the PATH). 

Solution: 

I created a copy of the module which appends the needed paths to the various environment variables (instead of prepending). Then the systems bunzip2 used and the shared object is still available.
